src/Compiler/FSStrings.resx

```diff
@@ -220,7 +220,7 @@
     <value>A coercion from the value type \n    {0}    \nto the type \n    {1}    \nwill involve boxing. Consider using 'box' instead</value>
   </data>
   <data name="TypeIsImplicitlyAbstract" xml:space="preserve">
-    <value>This type is 'abstract' since some abstract members have not been given an implementation. If this is intentional then add the '[&lt;AbstractClass&gt;]' attribute to your type.</value>
+    <value>Non-abstract classes cannot contain abstract members. If this is intentional then provide a default member implementation or add the '[&lt;AbstractClass&gt;]' attribute to your type.</value>
```

Comment:
"If this is intentional" - what is intentional? :) 

I think this can be just:
```suggestion
    <value>Non-abstract classes cannot contain abstract members. Either provide a default member implementation or add the '[&lt;AbstractClass&gt;]' attribute to your type.</value>
```

Reply:
Agreed. Will update it.